src/js/components/stock-movement-wizard/request/AddItemsPage.jsx

```diff
@@ -514,12 +655,23 @@ class AddItemsPage extends Component {
     }
   }
 
+  getIsRequestFromWard() {
+    this.setState({
+      isRequestFromWard: this.props.currentLocationId === this.state.values.destination.id && this.state.values.destination.type === 'WARD',
```

Comment:
This should be done in the `fetchAddItemsPageData` instead of in a separate method like this.

Reply:
done